Oqtane.Server/Infrastructure/LogManager.cs

```diff
@@ -95,27 +95,29 @@ private Log ProcessStructuredLog(Log Log)
                         names.Add(message.Substring(index + 1, message.IndexOf("}", index) - index - 1));
                         if (values.Length > (names.Count - 1))
                         {
-                            message = message.Replace("{" + names[names.Count - 1] + "}", values[names.Count - 1]?.ToString() ?? "null");
+                            if (values[names.Count - 1] == null)
```

Comment:
Why you prefer an "IF" instead of "??" operator, readability or technical reason?